src/main/java/uk/gov/hmcts/reform/idam/service/remote/client/IdamClient.java

```diff
@@ -19,13 +20,19 @@
 public interface IdamClient {
 
     @GetMapping(value = Constants.STALE_USERS_PATH, consumes = "application/json", produces = "application/json")
-    StaleUsersResponse getStaleUsers(@RequestParam Map<String, Object> queryParams);
+    StaleUsersResponse getStaleUsers(
+        @RequestHeader("Authorization") String authHeader,
+        @RequestParam Map<String, Object> queryParams
+    );
 
     @DeleteMapping(
             value = Constants.STALE_USERS_DELETE_PATH + "/{userId}",
```

Comment:
String STALE_USERS_DELETE_PATH is not required (again sorry for the confusion), we can use STALE_USERS_PATH instead.

Reply:
Updated.